scss/tech.scss

```
@@ -0,0 +1,44 @@
+.tech {
+    background: url('./assets/react_icon.svg') no-repeat;
+    background-color: $green;
+    background-position-x: center;
+    background-size: $tech-logo-size;
+    padding-bottom :30px;
+    padding-top :150px;
+    position: relative;
+    margin: 30px;
+    background-position-y: 30px;
+    border-radius: 10px;
+    width: 220px;
+ 
+    .text {
+      color: $equalizer-black;
+      display: flex;
+      font-size: $text-medium;
+      justify-content: center;  
+    }
+
+    #titleTech {
+      font-size: $text-medium;
+      padding-bottom: 10px;
+    }
+
+    #bold {
+      font-weight: bold;
+      font-size: $text-medium;
+      padding-bottom: 35px;
+    }
+
+    &:before{
+      content: "";
+      border-color: $green;
+      border-radius: 10px;
+      border-style: solid;
```

You can use a shorthand here
```suggestion
      border: 2px solid $green;
```